kernel/Kconfig

```diff
@@ -1,6 +1,7 @@
 config KSU
 	tristate "KernelSU module"
 	default y
+	depends on MODULES
```

Comment:
I don't think this is needed

Reply:
You said makes sense, I'll revert these commits.Maybe we just need to tell readers about the KPROBES dependencies on the website and let them debug by themselves.